### ISSUE TITLE
Reports block missing in admin panel

### ISSUE THREAD

**Issue by safi-manar** [WeblateOrg/weblate#1591]
### Steps to reproduce
1. Upgrade to version 2.16, currently: 479ae8bd3b3c64d0c7498ba59fb830864b895881

### Actual behaviour
The reports block (Status of repositories, SSH keys, Performance report) is not available in the admin panel. As a result, I cannot add SSH keys.
Going to /admin/ssh returns "Page Not Found The page you are looking for was not found."

### Expected behaviour
The reports block should be visible, as in version 2.15.

### Server configuration
 * Weblate weblate-2.15-346-g479ae8b
 * Python 2.7.13
 * Django 1.11.3
 * six 1.10.0
 * social-auth-core 1.4.0
 * social-auth-app-django 1.2.0
 * django-appconf 1.0.2
 * Translate Toolkit 2.2.4
 * Whoosh 2.7.4
 * defusedxml 0.5.0
 * Git 2.7.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.6.1
 * lxml 3.5.0
 * django-crispy-forms 1.6.1
 * compressor 2.1.1
 * djangorestframework 3.6.3
 * pytz 2017.2
 * pyuca N/A
 * python-bidi 0.4.0
 * pyLibravatar N/A
 * PyYAML 3.12
 * Database backends: django.db.backends.mysql

**Comment by Dchau95** [comment]
I think the error is due to routing issues. I compared code from 2.15 and 2.16 and saw that you when you added the get_url function in wladmin/sites.py, you kept admin/report, admin/ssh, and admin/performance which is not the correct url routes. You should remove the admin portion.

I tested this by going into django shell, creating the SITE variable, and seeing what get_urls returns and saw this output:

`[<RegexURLPattern index ^$>, <RegexURLPattern login ^login/$>, <RegexURLPattern logout ^logout/$>, <RegexURLPattern password_change ^password_change/$>, <RegexURLPattern password_change_done ^password_change/done/$>, <RegexURLPattern jsi18n ^jsi18n/$>, <RegexURLPattern view_on_site ^r/(?P<content_type_id>\d+)/(?P<object_id>.+)/$>, <RegexURLResolver <RegexURLPattern list> (None:None) ^accounts/profile/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^screenshots/screenshot/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^sites/site/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^permissions/autogroup/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^trans/componentlist/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^auth/user/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^social_django/nonce/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^social_django/usersocialauth/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^accounts/verifiedemail/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^trans/subproject/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^trans/whiteboardmessage/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^trans/project/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^trans/advertisement/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^social_django/association/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^permissions/groupacl/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^accounts/auditlog/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^lang/language/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^authtoken/token/>, <RegexURLResolver <RegexURLPattern list> (None:None) ^auth/group/>, <RegexURLPattern app_list ^(?P<app_label>accounts|screenshots|sites|permissions|trans|auth|social_django|lang|authtoken)/$>, <RegexURLPattern report ^admin/report/$>, <RegexURLPattern ssh ^admin/ssh/$>, <RegexURLPattern performance ^admin/performance/$>]`

What this output led me to is that, the correct url structure is /admin/admin/performance, which I double checked was the correct url route. This also led me to a server error due to the application not finding the admin/performance.html template which I think is due to wrong path issues as well.

TL;DR I think the issue above can be fixed by re-evaluating the url routes/project paths and double checking if they lead to the correct pages.

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/admin/upgrade.html#upgrade-from-2-15-to-2-16, you need to add wladmin to installed app to make that work. 

The double admin part in the URL is indeed not intentional, but the box would just appear with those links if the app is added.
